src/client/src/orangehrmCorporateDirectoryPlugin/components/EmployeeDetails.vue

```diff
@@ -0,0 +1,224 @@
+      </a>
+    </div>
+  </div>
+  <div
+    class="orangehrm-directory-card-hover"
+    @mouseleave="showTelephoneClip = false"
+    @mouseover="showTelephoneClip = true"
+  >
+    <div class="orangehrm-directory-card-hover-body">
+      <oxd-text type="toast-message">{{ $t('Work Telephone') }}</oxd-text>
+      <oxd-text ref="cloneTelephone" type="toast-title">
+        {{ employeeWorkTelephone }}
+      </oxd-text>
+    </div>
+    <div
+      class="orangehrm-directory-card-hover-body orangehrm-directory-card-icon"
+    >
+      <oxd-icon-button
+        v-show="showTelephoneClip"
+        name="clipboard-check"
+        @click="copyTelephone"
+      ></oxd-icon-button>
+    </div>
+  </div>
+  <oxd-divider></oxd-divider>
+  <div
+    class="orangehrm-directory-card-hover"
+    @mouseleave="showEmailClip = false"
+    @mouseover="showEmailClip = true"
+  >
+    <div class="orangehrm-directory-card-hover-body">
+      <oxd-text type="toast-message">{{ $t('Work Email') }}</oxd-text>
+      <oxd-text ref="cloneEmail" type="toast-title">
+        {{ employeeWorkEmail }}
+      </oxd-text>
+    </div>
+    <div
+      class="orangehrm-directory-card-hover-body orangehrm-directory-card-icon"
+    >
+      <oxd-icon-button
+        v-show="showEmailClip"
+        name="clipboard-check"
+        @click="copyEmail"
+      ></oxd-icon-button>
+    </div>
+  </div>
+  <oxd-divider></oxd-divider>
+</template>
+
+<script>
+import OxdDivider from '@ohrm/oxd/core/components/Divider/Divider';
+import {APIService} from '@/core/util/services/api.service';
+
+export default {
+  name: 'EmployeeDetails',
+  components: {
+    'oxd-divider': OxdDivider,
+  },
+  props: {
+    employeeId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup() {
+    const http = new APIService(
+      'https://07bd2c2f-bd2b-4a9f-97c7-cb744a96e0f8.mock.pstmn.io',
+      'api/v2/corporate-directory/employees',
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      employeeWorkTelephone: null,
+      employeeWorkEmail: null,
+      showTelephoneClip: false,
+      showEmailClip: false,
+    };
+  },
+  beforeMount() {
+    this.http.get(this.employeeId).then(response => {
+      const {data} = response.data;
+      this.employeeWorkTelephone = data.contactInfo?.workTelephone;
+      this.employeeWorkEmail = data.contactInfo?.workEmail;
+    });
+  },
+  methods: {
+    copyEmail() {
+      navigator.clipboard?.writeText(this.employeeWorkEmail);
+    },
+    copyTelephone() {
+      navigator.clipboard?.writeText(this.employeeWorkTelephone);
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.orangehrm-directory-card {
+  &-rounded-body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-right: 1rem;
+    padding-left: 1rem;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    border-radius: 100px;
+    width: 140px;
+    height: 64px;
+    box-shadow: 5px 5px 5px 5px #fafafc;
```

Comment:
```suggestion
    box-shadow: 5px 5px 5px 5px $color??;
```
@Super-Chama @KavindaK7 please check this color

Reply:
done